src/main/java/com/itschool/eventmanagment/controllers/EventController.java

```diff
@@ -49,4 +52,17 @@ public ResponseEntity<List<EventDTO>> getFilteredAndSortedEvents(@RequestParam @
                                                                      @RequestParam(required = false) String sortBy) {
         return ResponseEntity.ok(eventService.getEvents(to, from, location, sortBy));
     }
+    @DeleteMapping("/{id}")
+    public ResponseEntity deleteEvent(@PathVariable("id") Long id){
+        eventService.deleteEvent(id);
+        return ResponseEntity.ok("Event delete succesfully");
```

Comment:
We can consider `delete` methods as void, because usually the caller can rely only on the HTTP status code, not on the message we return as a response. In this case, can use [204 No Content](https://www.akto.io/academy/204-status-code).